Gemfile.lock

```diff
@@ -310,7 +310,7 @@ GEM
       listen (~> 2.7, >= 2.7.3)
     retriable (1.4.1)
     retryable (1.3.5)
-    rmagick (2.13.2)
+    rmagick (2.13.4)
```

Comment:
`bundle update rmagick` installed rmagick (2.15.2) for me.  Is 2.13.4 sufficient here?